### ISSUE TITLE
[extesion/sumologic] Add information about re-registration and eventually clobber being required for changing time_Zone and other properties 

### ISSUE THREAD

**Issue by sumo-drosiek** [SumoLogic/sumologic-otel-collector#740]
There is no way to update collector properties like tag or time_zone without re-registration (with clobber to keep same collector name), but there is no such info in our documentation

**Comment by mwtilton** [comment]
Yes this would be nice as it becomes quite cumbersome to fix at scale.

It would be nice to have an installation option for the MSI package as well. Similar to how #1437 added `-RemotelyManaged` flag to the installation script.

I'm also guessing this could be considered against the intended purpose of the `otel` collector, given it is deemed to be poor for remote management, but the `installed` collector is not much better.

https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/comparison.md
> are looking for a single agent as opposed to managing multiple agents

With the new `-RemotelyManaged` flag, it would seem to indicate that this is no longer an issue, but I think some of the underlying configuration changes need to catch up to this new change.
